release-notes/release-notes5101.md

```diff
@@ -66,7 +66,9 @@ For information on:
 
 ### Security related dependency update for all 3 server-side services
 
-All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+> **Important**: These security issues have not been resolved in the 5.10.1 release and will be resolved in an upcoming release. Note that no details of the security vulnerabilities have been publicly disclosed at the time of release.
+
+~~All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.~~
```

Comment:
https://ephocks.atlassian.net/browse/CLOUD-659 describes a DoS vulnerability in netty. Our spelling service is trivial to DoS without netty's help, so I'd describe the impact as "low".

I didn't pay enough attention to the release notes first time around, but I'd probably downplay this a bit now. The fixes are for https://nvd.nist.gov/vuln/detail/CVE-2021-37136 & https://nvd.nist.gov/vuln/detail/CVE-2021-37137 and were initially assessed as high due in line with the CVE classification, but further investigation considers them less serious in our context. We've also discovered that some dependencies remained unpatched by 5.10.1, and will be resolving those in an upcoming release.